src/tooltip/tooltip.js

```diff
@@ -284,15 +280,16 @@ angular.module( 'ui.bootstrap.tooltip', [ 'ui.bootstrap.position', 'ui.bootstrap
             /**
              * Observe the relevant attributes.
              */
-            if (!options.useContentExp) {
-              attrs.$observe( type, function ( val ) {
+            attrs.$observe( type, function ( val ) {
+              $timeout(positionTooltip, 0, false);
+              if (!options.useContentExp) {
```

Comment:
I moved the check inside the $observe because now it's also running positionTooltip inside the $observe independently of the check. This saves us from making two $observes on the same attr.